site/docs/master/api-types/backupstoragelocation.md

```diff
@@ -40,6 +40,7 @@ The configurable parameters are as follows:
 | `config` | map[string]string | None (Optional) | Provider-specific configuration keys/values to be passed to the object store plugin. See [your object storage provider's plugin documentation][0] for details. |
 | `accessMode` | String | `ReadWrite` | How Velero can access the backup storage location. Valid values are `ReadWrite`, `ReadOnly`. |
 | `backupSyncPeriod` | metav1.Duration | Optional Field | How frequently Velero should synchronize backups in object storage. Default is Velero's server backup sync period. Set this to `0s` to disable sync. |
+| `validationFrequency` | metav1.Duration | Optional Field | How frequently Velero should validate the object storage . Default is Velero's server validation frequency. Set this to `0s` to disable validation. |
```

Comment:
To disable, should this be `0s` or can it be just `0`?

Reply:
I _think_ `time.Duration` values need a unit at the end, so `0s`.

Reply:
Same as `backupSyncPeriod` right above.